### PR TITLE
[1LP][RFR] New HA test and clean up/fixed old tests for ap+cli

### PR DESCRIPTION
**Pull request by lcouzens**
Added additional testing for appliance_console and appliance_console_cli aimed at HA configurations. Cleaned up older tests to only run on the current build version, removed old support. Also cleaned up collections and skipping tests.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0c7cbdee7247b4010309adfc6aacc70d17eb7bb0:

`cfme/fixtures/cli.py`:
- :bangbang: Line 69:1: **E302** *expected 2 blank lines, found 1*

`cfme/tests/cli/test_appliance_cli.py`:
- :red_circle: Line 107:26: **F821** *undefined name 'BZ'*
- :red_circle: Line 112:5: **F821** *undefined name 'app1'*
- :red_circle: Line 131:75: **F821** *undefined name 'app1_ip'*

`cfme/tests/cli/test_appliance_console.py`:
- :bangbang: Line 201:61: **E231** *missing whitespace after ','*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 55a74e0986380f19c834615382266e326b0d4148:

`cfme/fixtures/cli.py`:
- :bangbang: Line 69:1: **E302** *expected 2 blank lines, found 1*

`cfme/tests/cli/test_appliance_console.py`:
- :bangbang: Line 201:61: **E231** *missing whitespace after ','*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4f56c99882bced242f2317b3a6206e2b2b2f1c14:

`cfme/fixtures/cli.py`:
- :bangbang: Line 65:1: **E303** *too many blank lines (3)*
- :bangbang: Line 79:1: **E303** *too many blank lines (3)*

`cfme/tests/cli/test_appliance_cli.py`:
- :heavy_exclamation_mark: Line 152:29: **W292** *no newline at end of file*

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 209:5: **E303** *too many blank lines (2)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 98d266b901c9af25e9dbd26d85da7ee4caf9f8fd:

`cfme/tests/cli/test_appliance_cli.py`:
- :red_circle: Line 3:1: **F401** *'cfme.utils.blockers.BZ' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by lcouzens** [comment]
Failures are provision related, no templates available.

File: cfme/tests/cli/test_appliance_cli.py [hunk 1]
```diff
@@ -80,24 +80,71 @@ def test_appliance_console_cli_ipa_crud(ipa_creds, configured_appliance):
...
+    apps[1].appliance_console_cli.configure_appliance_dedicated_ha_standby(
+        app_creds['username'], app_creds['password'], 'standby', app0_ip, app1_ip, '2',
+        'vmdb_production', apps[1].unpartitioned_disks[0]
+    )
+    # Configure automatic failover on EVM appliance
+    command_set = ('ap', '', '9', '1', '')
+    apps[2].appliance_console.run_commands(command_set)
+
+    def is_ha_monitor_started(appliance):
+        assert appliance.ssh_client.run_command(
```
Comment (quarckster) [inline cfme/tests/cli/test_appliance_cli.py]: **REQUIRED**
```python
def is_ha_monitor_started(appliance):
    return bool(appliance.ssh_client.run_command("grep {} /var/www/miq/vmdb/config/failover_databases.yml".format(app1_ip)).success)

wait_for(is_ha_monitor_started, func_args=[apps[2]], timeout=300)

File: cfme/tests/cli/test_appliance_cli.py [hunk 2]
```diff
@@ -80,24 +80,71 @@ def test_appliance_console_cli_ipa_crud(ipa_creds, configured_appliance):
...
+            "cat /var/www/miq/vmdb/config/failover_databases.yml | grep {}".format(app1_ip))
+    wait_for(is_ha_monitor_started, func_args=[apps[2]], timeout=300, handle_exception=True)
+    # Cause failover to occur
+    rc, out = apps[0].ssh_client.run_command('systemctl stop $APPLIANCE_PG_SERVICE', timeout=15)
```
Comment (quarckster) [inline cfme/tests/cli/test_appliance_cli.py]: **OPTIONAL**
If a variable will not be used further you should name it as `__`
```python
rc, __ = apps[0].ssh_client.run_command('systemctl stop $APPLIANCE_PG_SERVICE', timeout=15)
```

**Pull request merged**